backend/src/user.rs

```diff
@@ -54,12 +56,28 @@ fn get_user_data(
         None => None,
     };
 
-    let (posts, older_date, newer_date) = get_posts_for_latest_or_given_date(conn, user.id, date)?;
+    let (posts, older_date, newer_date, today_date) = get_posts_for_latest_or_given_date(conn, user.id, date)?;
```

Comment:
```suggestion
    let (posts, older_date, newer_date) = get_posts_for_latest_or_given_date(conn, user.id, date)?;
    let today_date = ft_sdk::env::now();
```

Don't pass it from function.